### ISSUE TITLE
Pain in penis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10099]
**Preferred term label:**
Pain in penis

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0497481

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ischemic priapism

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000064
- label: Pain in penis
- parents: [Pain (HP:0012531)](https://hpo.jax.org/app/browse/term/HP:0012531)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the penis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10099
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
HPO ID corrected to  HP:0040411
